### PR TITLE
Add German translations

### PR DESCRIPTION
**Pull request by siggi1984**
Add German language

### CODE REVIEW

**Comment by 25huizengek1** [comment]
Seems good, but that is unfortunately not how you translate in Android. You’ll have to add overrides for German. That way the default language is still present and untranslated strings will default to that. 

What I am saying is that if I’d merge right now, all of the English would be removed.

For more information on how to translate for Android, please have a look at this: https://developer.android.com/guide/topics/resources/localization#creating-alternatives

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -8,302 +8,302 @@
...
 
-    <string name="format_go_to_artist">More from %1$s</string>
+    <string name="format_go_to_artist">Mehr von %1$s</string>
     <string name="format_top_playlist">Top %1$s</string>
-    <string name="format_my_top_playlist">My top %1$s</string>
+    <string name="format_my_top_playlist">Meine Top %1$s</string>
     <string name="format_view_top_of_header">View top %1$s of …</string>
     <string name="format_speed_multiplier">%1$sx</string>
-    <string name="format_opening_url">Opening url %1$s</string>
+    <string name="format_opening_url">Öffne url %1$s</string>
```
Comment (25huizengek1) [inline app/src/main/res/values/strings.xml:16]: I’m not native German, but I think this should be in a continuous form?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -8,302 +8,302 @@
...
-    <string name="remove_from_queue">Remove from queue</string>
-    <string name="remove_from_playlist">Remove from playlist</string>
+    <string name="remove_from_queue">Aus der Warteschlange entfernen</string>
+    <string name="remove_from_playlist">Aus der Wiedergabeliste entfernen</string>
 
-    <string name="hide_from_quick_picks">Hide from \"Quick picks\"</string>
-    <string name="confirm_hide_song">Do you really want to hide this song? Its playback time and cache will be wiped.\nThis action is irreversible.</string>
-    <string name="confirm_delete_playlist">Do you really want to delete this playlist?</string>
+    <string name="hide_from_quick_picks">Ausblenden von \"Schnell-Auswahl\"</string>
+    <string name="confirm_hide_song">DMöchten Sie dieses Lied wirklich ausblenden? Die Wiedergabezeit und der Cache werden gelöscht. Diese Aktion ist nicht umkehrbar.</string>
```
Comment (25huizengek1) [inline app/src/main/res/values/strings.xml:97]: Typo?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -8,302 +8,302 @@
...
-    <string name="backup_action_description">Export the database to the external storage</string>
-    <string name="restore">Restore</string>
-    <string name="restore_warning">Existing data will be overwritten.\nViMusic will automatically close itself after restoring the database.</string>
-    <string name="restore_description">Import the database from the external storage</string>
-
-    <string name="other">Other</string>
+    <string name="previous_button_while_collapsed">Vorherige Taste, wenn sie zugeklappt ist</string>
+    <string name="previous_button_while_collapsed_description">Zeigt die Schaltfläche für den vorherigen Titel an, wenn der Player zusammengeklappt ist.</string>
+    <string name="swipe_horizontally_to_close">Horizontal wischen zum Schließen</string>
+    <string name="swipe_horizontally_to_close_description">Schließt den Player, wenn Sie auf dem eingeklappten Player nach links/rechts wischen. Nützlich für Benutzer, bei denen der Einhandmodus von Android\ aktiviert ist.</string>
```
Comment (25huizengek1) [inline app/src/main/res/values/strings.xml:214]: ‘\’ escapes nothing here, should be removed?

**Review by 25huizengek1 — CHANGES_REQUESTED** [review]
Overall good translations, I think, I’m not particularly fluent in German though, but I’ve added a few comments.

**Comment by siggi1984** [comment]
> Seems good, but that is unfortunately not how you translate in Android. You’ll have to add overrides for German. That way the default language is still present and untranslated strings will default to that.
> 
> What I am saying is that if I’d merge right now, all of the English would be removed.
> 
> For more information on how to translate for Android, please have a look at this: https://developer.android.com/guide/topics/resources/localization#creating-alternatives

Oh sorry, I will try to correct it. The typos too, of course.

**Comment by 25huizengek1** [comment]
No problem. Another tip: to cover all German regions / countries you should add a `values-de` directory.

What I meant with regions, these are available in Android:
image

It should look like this:
image

Please only include the strings that are translated, including the strings that stay the same, but not the strings that are still in English. If you are using Android Studio, you could also use the translations editor, as suggested in the screenshot.

**Comment by siggi1984** [comment]
I have translated and corrected the strings file. Unfortunately I can't manage to create the folder values-de at github to put the file in there. Can I send you the file and you put it in the folder. I assume that it would not be a problem for you.

**Comment by siggi1984** [comment]
Okay, closed this and redid everything once. Hope everything is correct this time.

**Pull request closed without merging**